bridges/evm/contracts/MultiStore.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity 0.5.14;
+import {Utils} from "./Utils.sol";
+
+
+// Computes Tendermint's application state hash at this given block. AppHash is actually a
+// Merkle hash on muliple stores. Luckily, we only care about "oracle" tree and all other
```

Comment:
This is not true anymore. We are not that lucky. Please read comment before copy-pasting